src/test/suites/interface-suite.js

```diff
@@ -186,17 +186,33 @@ suite.declare((lab, variables) => {
       return done();
     });
 
-    lab.test("A null source field throws an error", done => {
+    lab.test("A null source field should map the source to destination", done => {
```

Comment:
The most common use case will be ```map().to("whatever")``` not ```map(null).to("whatever")``` but we are not testing it that way. Probably should have an additional test for that.